core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3462,7 +3491,8 @@ paths:
         - billing
         - users
       summary: billing project status
-      description: billing project status
+      description: Billing project status. Deprecated, use v2 API.
```

Comment:
We don't have a specific "status" endpoint in v2, but status is included in the billing project response.